docs/index.rst

```diff
@@ -3,6 +3,370 @@
 HDFS Connector
 ==============
 
-The HDFS connector allows you to export data from Kafka topics to HDFS in a
-variety of formats and integrates with Hive to make data immediately available
-for querying with HiveQL.
+The HDFS connector allows you to export data from Kafka topics to HDFS files in a variety of formats
+and integrates with Hive to make data immediately available for querying with HiveQL.
+
+Quickstart
+----------
+In this Quickstart, we use the HDFS connector to export data produced by the Avro console producer
+to HDFS.
+
+Start Zookeeper, Kafka and SchemaRegistry is you haven't done so. The instructions on how to start
+these services is available in this document. You also need to have Hadoop running locally or
+remotely and makes sure that you know the HDFS url. For Hive integration, you also need to have
+Hive installed. This Quickstart assumes that you started the required services with the default
+config and you should make necessary changes according to the actual configuration used.
+
+First, start the Avro console producer::
+
+  $ ./bin/kafka-avro-console-producer --broker-list localhost:9092 --topic test_hdfs \
+  --property value.schema='{"type":"record","name":"myrecord","fields":[{"name":"f1","type":"string"}]}'
+
+Then in the console producer, type in::
+
+  {"f1": "value1"}
+  {"f1": "value2"}
+  {"f1": "value3"}
+
+The three records entered will be published to the Kafka topic ``test_hdfs`` in Avro format.
+
+Next, in ``./config/connect-standalone.properties``, we need to use
```

Comment:
The should probably be etc/schema-registry/connect-avro-standalone.properties
./config/... doesn't exist in our packages